composer.json

```diff
@@ -21,7 +21,7 @@
             "email": "zoohie@gmail.com"
         }
     ],
-    "minimum-stability": "dev",
+    "minimum-stability": "beta",
```

Comment:
Alternatively we could set minimum-stability stable and just pin to amphp/http-client v5.0.0-beta.11 (this strategy won't work for transitive beta/dev dependencies but all of amphp/http-client's dependencies are stable now)

Reply:
Removing and adding -beta to http client sounds good to me.